File: program1/main.go
package main

import (
	"fmt"
)

func main() {
	var n float64
	fmt.Print("Enter number: ")
	_, err := fmt.Scanf("%f", &n)
	if err != nil {
		fmt.Println("Invalid input")
		return
	}

	if (n > 0 && n < 10) || (n < 0 && n > -10) {
		fmt.Printf("%.2f is a 1-digit number\n", n)
	} else if (n >= 10 && n <= 99) || (n <= -10 && n >= -99) {
		fmt.Printf("%.2f is a 2-digit number\n", n)
	} else if (n > 99 && n <= 999) || (n < -99 && n >= -999) {
		fmt.Printf("%.2f is a 3-digit number\n", n)
	} else {
		fmt.Println("Invalid Case")
	}
}

File: program1/main.py
n=float(input("Enter number: "))
if((n>0 and n <10) or (n<0 and n>-10)):
    print(f"{n} is a 1-digit number")
elif((n>=10 and n <=99) or (n<=-10 and n>=-99)):
    print(f"{n} is a 2-digit number")
elif((n>99 and n<=999) or (n<-99 and n>= -999)):
    print(f"{n} is a 3-digit number")
else:
    print("Invalid Case")
